sympy/ntheory/digits.py

```diff
@@ -111,16 +111,21 @@ def count_digits(n, b=10):
 
 def is_palindromic(n, b=10):
     """return True if ``n`` is the same when read from left to right
-    or right to left in the given base, ``b``.
+    or right to left in the given base, ``b``.(b must be greater than 1)
 
     Examples
     ========
 
     >>> from sympy.ntheory import is_palindromic
 
-    >>> all(is_palindromic(i) for i in (-11, 1, 22, 121))
+    >>> all(is_palindromic(i) for i in (11, 1, 22, 121))
     True
 
+    Note:negative integers can't be palindromic by definition
```

Comment:
I have been indecisive about this how such definition should restrict negative integers,
But if it is because of the simple reason that `-` sign moves to right with the string reversal, it should be noted in documentation.